assets/src/edit-story/components/library/panes/text/karma/textPane.karma.js

```diff
@@ -32,6 +41,10 @@ describe('CUJ: Creator can Add and Write Text: Consecutive text presets', () =>
 
   it('should add text presets below each other if added consecutively', async () => {
     await fixture.editor.library.textTab.click();
+    // @todo Remove this once #4094 gets fixed!
+    // Wait until the history has changed to its initial (incorrect due to a bug) position.
+    await fixture.events.sleep(300);
+
     await fixture.events.click(fixture.editor.library.text.preset('Heading 1'));
     await fixture.events.click(fixture.editor.library.text.preset('Heading 3'));
     await fixture.events.click(fixture.editor.library.text.preset('Paragraph'));
```

Comment:
Are we sure these are added correctly?

Reply:
From tests:
<img width="383" alt="Screenshot 2020-09-03 at 18 43 35" src="https://user-images.githubusercontent.com/3294597/92137033-0428f500-ee0d-11ea-9f5e-2acb337c9a88.png">